### DISCUSSION TITLE
Notification with incorrect description

### DISCUSSION THREAD

**Discussion by schrej** [community/community#29295] [Mobile]
I think there is a small bug in the notification list in the Android app.

I have a notification that claims that a review was requested from me. But actually a review was requested from someone else.

![Screenshot_20220812-192152.png](https://user-images.githubusercontent.com/1710904/184411537-c8c19daf-9a42-4a05-948b-f12cc9a1e4f2.png)

The PR: https://github.com/metal3-io/cluster-api-provider-metal3/pull/678

Version: 1.76.0
Latest Android 12 on Google Pixel 5

**Comment by haldun** [comment]
Hi @schrej, thanks for reaching out and for the bug report. I will pass this on to the team to investigate.
